.eslintrc.js

```diff
@@ -10,42 +12,45 @@ module.exports = {
   overrides: [
     {
       files: [
-        'public/js/**/*.js'
+        "public/js/**/*.js",
       ],
       env: {
         browser: true,
-        node: false
+        node: false,
       },
       rules: {
-        "strict": "off"
-      }
+        "strict": "off",
+      },
     },
     {
       files: [
-        'scripts/*.js',
+        "scripts/*.js",
       ],
       rules: {
         "no-console": "off",
         "no-process-exit": "off",
-      }
+      },
     },
     {
       files: [
-        'tests/**/*.js',
+        "tests/**/*.js",
       ],
       env: {
         jest: true,
       },
     },
     {
       files: [
-        'tests/integration/**/*.js',
+        "tests/integration/**/*.js",
       ],
       globals: {
         "$": "readonly",
         "$$": "readonly",
         "browser": "readonly",
       },
+      rules: {
+        "valid-jsdoc": "off",
```

Comment:
Sorry, I don't know that I care about integration tests having properly formed JSDoc comments.
I think this was flagging some blocks on missing `@returns` meta.